src/lib/operators/join_sort_merge/radix_cluster_sort.hpp

```diff
@@ -56,7 +56,7 @@ class RadixClusterSort {
         _right_input_table{right},
         _left_column_id{column_ids.first},
         _right_column_id{column_ids.second},
-        _sort{sort},
+        _sort_result_clusters{sort},
```

Comment:
I'd say either `_sort_resulting_clusters` or `_sort_clusters` (my favorite).

Reply:
I used `_sort_resulting_clusters` because we have already a function with the name  `_sort_clusters `.